packaging/windows/mk

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -eu
+VER=$(sed -nE "s/^version = \"(.+)\"\$$/\1/p" ../../bin/Cargo.toml)
```

Comment:
This means we're stuck with just the Major.Minor.Patch versions rather than full Major.Minor.Patch.Build versions, which may be insufficient for upgrade purposes if we do a rebuild to pick up something like a security patch to the compiler without bumping the patch version.

We have some logic over in the cdn build to handle this on the linux packages. https://github.com/answerbook/logdna-cdn/blob/48c23808f088c539c2d849c6ba2af6505149e0a3/deployment/docker/Dockerfile#L105

Reply:
Current MSI allows patch updates in any direction. Works.